tests/system/lab/test

```diff
@@ -897,6 +898,21 @@ else
      echo "======================= SKIPPED PI EGRESS ======================="
 fi
 
+echo "===================== COLLECTING SUPPORT BUNDLE ============================"
+support_bundle="$PROJECT_ROOT/support"
```

Comment:
I was asking to change the directory name NOT variable name.
`$PROJECT_ROOT/support_bundle`

Reply:
ok